sim_volumes.py

```diff
@@ -0,0 +1,172 @@
+"""Generate 3D map of molecules."""
+
+import numpy as np
+from geomstats.geometry.special_orthogonal import SpecialOrthogonal
+
+SO3 = SpecialOrthogonal(n=3, point_type="vector")
+
+PARTICULES = np.asarray([[1, 0, 0, 3], [0, 8, 0, 5], [0, 0, 7, 9]])
+N_VOLUMES = 2000
+VOL_SIZE = 64
+NAME = "4_points_3d"
+CENTER = 2
+
+
+def get_random_quat(num_pts):
+    """Generate a list of quaternions by using uniform distribution.
+
+    Parameters
+    ----------
+    num_pts : int
+        number of quaternions to return.
```

Comment:
number -> Number

Reply:
And same with all parameters descriptions in `Parameters` and `Returns`:
description starts with uppercase and ends with a .